src/esp/core/esp.h

```diff
@@ -30,8 +30,14 @@ typedef Matrix<uint32_t, 4, 1> Vector4ui;
 typedef Matrix<uint64_t, 4, 1> Vector4ul;
 
 //! Eigen JSON string format specification
-static const IOFormat
-    kJsonFormat(StreamPrecision, DontAlignCols, ",", ",", "", "", "[", "]");
+static const IOFormat kJsonFormat(StreamPrecision,
+                                  DontAlignCols,
+                                  ",",   // coef separator
```

Comment:
```suggestion
                                  ", ",   // coef separator
```
Can I add a "space" behind the comma for the coef, and row separator? It looks better in my opinion.

The following is the one WITHOUT the space:
![image](https://user-images.githubusercontent.com/931093/64657113-cc989000-d3e6-11e9-8cee-96a8d011a15f.png)

And the one with the space:
![image](https://user-images.githubusercontent.com/931093/64657128-dcb06f80-d3e6-11e9-9b47-61081482595d.png)

Reply:
Please let me know your thoughts. Thanks.

Reply:
Looks good to me.